README.md

```diff
@@ -32,15 +32,20 @@ First since we'll launch kind with GPU support, ensure that the following prereq
    Container Runtime to use volume mounts to select devices to inject into a
    container.
 
+1. Show the current GPU information of the machine
+   ```bash
+   nvidia-smi -L
+   ```
+
 We start by first cloning this repository and `cd`ing into it.
 All of the scripts and example Pod specs used in this demo are in the `demo`
 subdirectory, so take a moment to browse through the various files and see
 what's available:
 
-```
+```bash
```

Comment:
I see bash/console should we agree on a single one? is there a benefit of having one vs the other on specific steps? (is one better for copy/pasting than the other for example?

Reply:
Changed all of them to `console`.